core/templates/dev/head/pages/exploration-editor-page/modal-templates/post-publish-modal.template.html

```diff
@@ -67,8 +66,10 @@ <h3>Awesome!</h3>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
     <div class="oppia-share-publish-link">
-      Here's your exploration link:
+      Here's your exploration link:<br>
+      <span style="visibility:hidden" ng-show="explorationLinkCopied">&nbsp Copied &nbsp<i class="fa fa-check" aria-hidden="true"></i></span>
```

Comment:
Let's remove this it makes it hard to understand the code, I think a better way would be to use `positioning` for  "Copied" element. What do you think?

Reply:
Sure @DubeySandeep, I committed using the positioning tag for Copied element.